README.md

```diff
@@ -360,6 +360,45 @@ See [the Optimizely REST API documentation](http://developers.optimizely.com/res
 # }
 ```
 
+###Get Experiment Results (Stats Engine)
+To list stats engine results, call stats on the associated `Experiment` object.
```

Comment:
nit: should be capital "Stats Engine". would also be great if this was a link to the Stats Engine documentation: https://help.optimizely.com/hc/en-us/articles/200039895-Stats-Engine-How-Optimizely-calculates-results-to-enable-business-decisions